IdentityCore/src/cache/crypto/MSIDAssymetricKeyKeychainGenerator.m

```diff
@@ -26,6 +26,8 @@
 #import "MSIDAssymetricKeyPair.h"
 #import "MSIDAssymetricKeyLookupAttributes.h"
 
+static const OSStatus NoStatus = -1;
```

Comment:
Can we change it to kNoStatus to follow our code style?

Reply:
absolutely